### PR TITLE
Restructuring the ui for mobile

### LINKED ISSUE #13: make responsive UI (ie mobile friendly)
## Problem
It turns out that the current layout is not responsive. We need to re-design layout from scratch using e.g. bootstrap or foundation or similar.

## Solution
- [ ] Rewrite markup and CSS with mobile-first approach
- [ ] Use bootstrap, foundation or similar for the above - and adapt it too look as specified / current layout

### LINKED ISSUE #44: Network graph overlapping footer
The network graph is currently overlapping the footer.
screen shot 2018-09-20 at 13 52 52

It also sometimes overlaps the header:
screen shot 2018-09-20 at 14 05 19

### PR DESCRIPTION
**Pull request by kelsos**
Closes #13 
Closes #44

### CODE REVIEW

File: angular-app/angular.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.css",
+              "src/status-theme.scss"
```
Comment (palango) [inline angular-app/angular.json]: We should gradually switch to the name **Explorer**, so please change that here.

File: angular-app/src/app/components/carousel/carousel.component.ts [hunk 2]
```diff
@@ -14,7 +25,7 @@ export class CarouselItemElementDirective {
   templateUrl: 'carousel.component.html',
   styleUrls: ['./carousel.component.css']
 })
-export class CarouselComponent implements AfterViewInit {
+export class CarouselComponent implements  OnInit {
```
Comment (palango) [inline angular-app/src/app/components/carousel/carousel.component.ts]: Extra space here

File: angular-app/src/app/components/home/home.component.html [hunk 3]
```diff
@@ -1,87 +1,157 @@
...
-          <a href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#channels" class="link-button">
+        <div fxLayout="column"
+             fxLayout.lt-md="row"
+             fxFlex="0 0 200px"
+             fxFlex.lt-md="0 0 auto">
+          <a mat-stroked-button
+             color="primary"
+             [ngStyle.lt-md]="{'width': '380px'}"
+             matTooltip="Learn how to open a channel"
+             href="http://raiden-network.readthedocs.io/en/stable/webui_tutorial.html#channels">
```
Comment (palango) [inline angular-app/src/app/components/home/home.component.html]: Loredana asked to make these links open in a new tab, can you add this: https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window

File: angular-app/angular.json (hunk 1 above)
Comment (kelsos) [reply angular-app/angular.json]: Should I rename status to explorer everywhere? or do it in the next PR?

File: angular-app/src/app/components/home/home.component.html (hunk 3 above)
Comment (kelsos) [reply angular-app/src/app/components/home/home.component.html]: I was planning on making them on the next PR :)

File: angular-app/angular.json (hunk 1 above)
Comment (palango) [reply angular-app/angular.json]: Fine in another PR as well.

**Pull request merged**